### DISCUSSION TITLE
Binary install for Arch based distributions

### DISCUSSION THREAD

**Discussion by medmedin2014** [senshu/Sozi#531] [Ideas]
The package Sozi in AUR takes really long time and requires many dependencies to be compiled and installed, currently most Arch packages are migrating to provide binary packages to make it easy to install and avoid any compilation on user machine. It would be cool to have this kind of package for Sozi plus the binary install is already provided for Linux.
Today I tried to install it on Manjaro KDE and it's really crazy install process with more than 10000 lines in [install log](https://drive.google.com/file/d/13KUk-ShUu4D02zAsAG69oMQZRiMex8dp/view?usp=sharing).

**Comment by aumouvantsillage** [comment]
Indeed, users should not have to compile Sozi and its dependencies unless they have specific reasons.

Currently, Debian and Redhat packages are available because there are packaging tools that integrate well in the current build process: [electron-installer-debian](https://github.com/electron-userland/electron-installer-debian) and [electron-installer-redhat](https://github.com/electron-userland/electron-installer-redhat).

Unfortunately, is seems no such tool exists for Archlinux. It would be great if someone could help find a solution.
